### PR TITLE
First attempt at get_stats19

### PR DESCRIPTION
**Pull request by layik**
#11

### CODE REVIEW

File: R/get.R [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#' Download, read and format STATS19 data in one function.
+#'
+#' @section Details:
+#' This function utilizes `dl_stats19` and `read_*` functions
+#' and retuns a df. The file downloaded would be for a specific year (e.g 2017).
+#'
+#' As this function uses `dl_stats19` function, it can download
+#' many MB of data so ensure you have a sufficient disk space.
+#' @seealso \code{\link{dl_stats19}}
```
Comment (Robinlovelace) [inline R/get.R]: I think this line would be better as:

```r
#' @seealso [dl_stats19()]
```

See, for example, https://github.com/tidyverse/ggplot2/search?q=seealso&unscoped_q=seealso

Heads-up @adamhsparks, roxygen2 allows markdown shortcuts for some documentation code - we've started using markdown so I think we should continue for consistency (pre-empting responses to your review).

File: R/get.R [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+#' @seealso \code{\link{read_accidents}}
```
Comment (Robinlovelace) [inline R/get.R]: Same comment as above.

File: R/get.R [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  dl_stats19(year = year,
+             type = type,
+             data_dir = data_dir,
+             file_name = file_name)
+  read_in = NULL
+  # what did the user want?
+  # TODO: extra checks to see if user
+  # made the right choice. Or leave it to
+  # respective read_* functions to check.
+  if(grepl(type, "vehicles",  ignore.case = TRUE)){
```
Comment (Robinlovelace) [inline R/get.R]: Looks good. I think if we refactor the `read_*()` functions, this can be simplified. For now looks great.

**Comment by layik** [comment]
Thanks Robin. Please hold approval for more commits. I just wanted to push to get these feedbacks/comments. 

The ask = T may need a bit of more work but yes I agree.

File: R/get.R (hunk 1 above)
Comment (layik) [reply R/get.R]: I tried both and for some reason as I saw no “link” to the definition I opted for the current. Will switch and push or feel free to do it Robin.

**Comment by layik** [comment]
As commit message states: mandatory "type" *should* remove concerns of user giving `type = "acc"` but then choosing some `_vehicles_` file from download list. If user only provides type, then `read_*` will deal with any potential issues.

**Comment by layik** [comment]
Anything else before its ready for review @Robinlovelace? and I will focus on the rest of the second useful review.

**Comment by layik** [comment]
Will fix that later

**Pull request merged**